feat(): Add langing page for third party apps (#1828)
questdb/questdb.io: docs/third-party-tools.md

---
title: Third Party Tools
description:
  QuestDB integrates well with a number of third party tools. This page lists
  some of the most popular integrations.
---

> QuestDB integrates well with a number of third-party tools. This page lists some
> of the most popular integrations:

Reviewer: ```suggestion
QuestDB integrates well with a number of third-party tools. 

The following is a list of our most popular integrations:
```

(Pedantic) 

* White space helps at-a-glance readability
* Trying to stay on 1 line before width break = guarantee brevity
* More direct pointing without hedge
